.env.example

```
@@ -239,4 +239,4 @@ SITE_SWITCHER_JS_HASH=
 #BEATMAPSET_UPLOAD_BONUS_PER_RANKED_MAX=2
 #BEATMAPSET_UPLOAD_ALLOWED_SUPPORTER=8
 #BEATMAPSET_UPLOAD_BONUS_PER_RANKED_SUPPORTER=1
-#BEATMAPSET_UPLOAD_BONUS_PER_RANKED_LIMIT_MAX=2
+#BEATMAPSET_UPLOAD_BONUS_PER_RANKED_SUPPORTER_MAX=2
```

Use `_SUPPORTER` suffix just like other variables? Also matches the actual config name.

And match sample value with the default.

```suggestion
#BEATMAPSET_UPLOAD_BONUS_PER_RANKED_MAX_SUPPORTER=12
```